### PR TITLE
ui improvements

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: qml/ScenarioLoader.qml [hunk 1]
```diff
@@ -23,6 +23,7 @@ ColumnLayout
 	signal closed()
 	property alias selectedScenarioIndex: scenarioList.currentIndex
 	property bool panelLoaded: false
+    property int rowbtnWidth: 100
```
Comment (arkpar) [inline qml/ScenarioLoader.qml]: indent

File: qml/Watchers.qml [hunk 2]
```diff
@@ -77,11 +77,11 @@ Rectangle
 	Rectangle {
 		color: selectedTxColor
 		anchors.fill: parent
-		anchors.margins: 10
+        anchors.margins: 10
```
Comment (arkpar) [inline qml/Watchers.qml]: indent

File: src/ClientModel.cpp [hunk 3]
```diff
@@ -99,7 +99,7 @@ void ClientModel::init(QString _dbpath)
 	if (m_dbpath.isEmpty())
 		m_client.reset(new MixClient(QStandardPaths::writableLocation(QStandardPaths::TempLocation).toStdString()));
 	else
-		m_client.reset(new MixClient(m_dbpath.toStdString()));
+		m_client.reset(new MixClient(QStandardPaths::writableLocation(QStandardPaths::TempLocation).toStdString() + "/" + m_dbpath.toStdString()));
```
Comment (arkpar) [inline src/ClientModel.cpp]: Probably no need to add m_dbpath here, better remove _dbpath altogether.
Also, tmp dir should be cleaned up in destructor

Comment (yann300) [inline src/ClientModel.cpp]: ok cleaned up added. 
We still need to be able to specify another path cause two instances of ClientModel can run at the same time.

**Pull request merged**